var/spack/repos/builtin/packages/qmcpack/package.py

```diff
@@ -57,31 +59,76 @@ class Qmcpack(CMakePackage):
     # variant('+mixed', default=True, when='+cuda', description="...")
 
     # conflicts
-    conflicts('+soa', when='+cuda')
-    conflicts('^openblas+ilp64')
-    conflicts('^intel-mkl+ilp64')
+    conflicts(
+        '+phdf5',
+        when='~mpi',
+        msg='Parallel collective I/O requires MPI-enabled QMCPACK.\n \
+        Please add \'~phdf5\' to the Spack install line for serial QMCPACK.'
```

Comment:
https://www.python.org/dev/peps/pep-0008/#string-quotes

> When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability.

Reply:
I can address this.